### ISSUE TITLE
Instructions for Heroku

### ISSUE THREAD

**Issue by sinak** [aortbals/southwest-checkin#43]
I got my own instance of this working Heroku by doing the following:

- Adding redistogo as an add-on to the Heroku app
- Running `heroku config:set REDIS_PROVIDER=REDISTOGO_URL` from the commandline to get sidekiq to load the redis add-on's URL.
- Adding `ruby '2.2.0'` to the gemfile.
- Adding `gem 'rails_12factor', group: :production` to the gemfile, and commenting out `config.logger = ActiveSupport ...` line from `config/environments/production.rb`
- Running `heroku config:set SITE_NAME='Southwest Checkin' SITE_URL=<HEROKU_URL> ASSET_HOST=<HEROKU_URL> MAILER_DEFAULT_FROM_EMAIL=<MY_EMAIL> MAILER_DEFAULT_REPLY_TO=<MY_EMAIL> DEPLOY_BRANCH=master DEPLOY_USER=deploy DEPLOY_PORT=22 MAILER_ADDRESS= MAILER_DOMAIN= MAILER_USERNAME= MAILER_PASSWORD= MAILER_DEFAULT_HOST= DEPLOY_DOMAIN= DEPLOY_TO= DEPLOY_REPOSITORY= AIRBRAKE_API_KEY= AIRBRAKE_HOST= AIRBRAKE_DEPLOY_NOTIFICATION=false` and replacing <HEROKU_URL> and <MY_EMAIL>. Not sure if that was necessary.
- Running `bundle install` to generate a new `gemfile.lock`
- Committing everything and pushing it to Heroku. 

My instance is running at https://southwest-sina.herokuapp.com. It won't send emails or do a lot of other things, but it works! 

Thank you so much for making this all open source.

**Comment by aortbals** [comment]
@sinak I would advise not running a public version of this. See the reason at https://chkmein.co/.

**Comment by wootwoot1234** [comment]
@sinak I'm on the ```bundle install``` step but I'm getting the following error.  

> Your Ruby version is 2.0.0, but your Gemfile specified 2.2.0

Do you know how to change my Ruby version?

**Comment by wootwoot1234** [comment]
I'm totally new to ruby but got past this step so I thought I would share.  I had to install a few things first.  I'm running OSX and using homebrew I used these commands to install the correct libraries:

```
brew install redis
brew install postgresql
brew install rbenv
brew install ruby-build
rbenv init
eval "$(rbenv init -)"
rbenv install 2.2.0
```

Then I had to close terminal and reopen it.

Then from the project root directory I ran:
```
gem install bundler
rbenv rehash
bundle install
git add .
git commit -am "make it better"
git push heroku master
```

I'm not totally there, I'm still getting errors.  See issue https://github.com/aortbals/southwest-checkin/issues/52 but I'm getting closer.  If I can get this working I'll submit a PR with instructions.
